apps/examples/src/examples/exploded/ExplodedExample.tsx

```diff
@@ -29,7 +30,7 @@ const defaultComponents = {
 //[2]
 export default function ExplodedExample() {
 	return (
-		<div className="tldraw__editor">
+		<TldrawWrapper style={{ width: '100%', height: '100%' }}>
```

Comment:
I used a style prop here to indicate to the reader that styles go on the wrapper component, not the editor component.